infobip-typescript-generator-extension-model-showcase/src/main/java/com/infobip/typescript/showcase/custom/simple/SimpleValidationTypeScriptFileGenerator.java

```diff
@@ -0,0 +1,52 @@
+package com.infobip.typescript.showcase.custom.simple;
+
+import com.infobip.typescript.CustomValidationSettings;
+import com.infobip.typescript.TypeScriptFileGenerator;
+import com.infobip.typescript.showcase.custom.complex.ComplexValidationTypeScriptFileGenerator;
+import com.infobip.typescript.showcase.custom.simple.validation.Foo;
+import com.infobip.typescript.showcase.hierarchy.exception.UncheckedURISyntaxException;
+import cz.habarta.typescript.generator.Input;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.URISyntaxException;
+import java.nio.file.*;
+import java.util.Collections;
+import java.util.List;
+
+public class SimpleValidationTypeScriptFileGenerator extends TypeScriptFileGenerator {
+
+    public SimpleValidationTypeScriptFileGenerator(Path basePath) {
+        super(basePath);
+    }
+
+    @Override
+    public Input getInput() {
+        return Input.from(Foo.class);
+    }
+
+    @Override
+    public Path outputFilePath(Path basePath) {
+        Path lib = basePath.getParent().getParent().resolve("dist");
+
+        try {
+            Files.createDirectories(lib);
```

Comment:
same as the other comment, try including all logic in try catch block

Reply:
done